ci.md

```diff
@@ -1,31 +0,0 @@
-# Commands to trigger ci pipeline
```

Comment:
Do you have documentation for Prow?

Reply:
No need special document, now easy to trigger them.
Updated dev guide: https://pingcap.github.io/tidb-dev-guide/get-started/write-and-run-unit-tests.html#running-tests-for-a-pull-request

Reply:
Have you documentation about how to manage the pipeline and add a new job into the pipeline?

Reply:
I will add later when post submit pipelines are refactored.

Reply:
Current pinned a guide issue [#pingcap-qe/ci/](https://github.com/PingCAP-QE/ci/issues/1693). I plan to transform it to `tidb-dev-guide` repo